packages/protocol/contracts/common/Accounts.sol

```diff
@@ -334,7 +334,9 @@ contract Accounts is
 
   /**
    * @notice Gets validator payment delegation settings.
-   * @return Beneficiary address and fraction of payment delegated.
+   * @param address of the validator.
```

Comment:
The first word after `@param` should be the name of the parameter. Also, this might not be the direct address of the validator, but a reference to an account in this Accounts smart contract - happy to chat more about this difference later.

```suggestion
   * @param account Account of the validator.
```